e2e/coverage-handlebars/transform-handlebars.js

```diff
@@ -5,16 +5,16 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+const {build} = require('@jridgewell/build-mapping');
+const {encodedMap, AnyMap} = require('@jridgewell/trace-mapping');
 const Handlebars = require('handlebars/dist/cjs/handlebars.js');
-const {SourceMapConsumer, SourceNode} = require('source-map');
+const dedent = require('string-dedent');
 
 exports.process = (code, filename) => {
   const pc = Handlebars.precompile(code, {srcName: filename});
-  const out = new SourceNode(null, null, null, [
-    'const Handlebars = require("handlebars/dist/cjs/handlebars.runtime.js");\n',
-    'module.exports = Handlebars.template(',
-    SourceNode.fromStringWithSourceMap(pc.code, new SourceMapConsumer(pc.map)),
-    ');\n',
-  ]).toStringWithSourceMap();
-  return {code: out.code, map: out.map.toString()};
+  const out = dedent(build)`
+    const Handlebars = require("handlebars/dist/cjs/handlebars.runtime.js");
+    module.exports = Handlebars.template(${pc});
+  `;
+  return {code: out.code, map: encodedMap(new AnyMap(out.map))};
```

Comment:
I just implemented a `normalize` function that will output a compatible source map without flattening (which should be much faster):
```suggestion
  return normalize(out);
```

Reply:
Nice! I can also remove the `trace-mapping` dep, then